app/views/checkout/edit.html.haml

```diff
@@ -23,15 +23,22 @@
   .sub-header.show-for-medium-down
     = render partial: "shopping_shared/order_cycles"
 
-  %accordion{"close-others" => "false"}
+  - if feature? :split_checkout, spree_current_user
```

Comment:
In terms of code organisation and managing this feature toggle over time, it might be good to separate this a bit more and start building the new checkout with a separate controller and separate routes so we can leave the current checkout untouched.

So for example the existing checkout would use `CheckoutController`, with views under `app/views/checkout/*`, but the new one could be (something like) `NewCheckoutController` with separate views under `app/views/new_checkout/*`. 

What do you think?

Reply:
I think it's a good idea, but i don't want to speak for @andrewpbrett . 
I'd be happy to change this (or maybe @andrewpbrett if he starts working on it), if it's ok for you both ;)

Reply:
Quick question: I'm wondering how can we separate controller/views with feature toggle?

Reply:
Yeah I like that idea @Matt-Yorkley - we're not that far from it with these existing commits in the views, calling them `app/views/checkout/split_checkout/*`. @jibees I'll push some commits to this branch if that's okay to show you what I'm thinking.

Reply:
Take a look at those last two commits, hopefully it's clear. I added a constraint to the routes file to redirect to a separate controller if the split checkout feature is enabled. Right now the SplitCheckoutController just subclasses CheckoutController, but we can override methods as needed in it. Rails magically knows that you want the `split_checkout/*` views so I separated the conditional into two separate views.

Reply:
Yeah, that's clear, thanks for pointing me that out! Didn't know that `constraints` concept, the Rails magic ;)

Regarding the URLs, I think it's exactly where we can then customize it no? Thus, URLs must changed, to be as described here: https://github.com/openfoodfoundation/openfoodnetwork/issues/7409#issue-857177939

What do you think?